webrtc/libjingle/xmpp/boshsocket.cc

```diff
@@ -155,7 +156,9 @@ namespace buzz {
 #ifdef FEATURE_ENABLE_SSL
     if (tls_ != buzz::TLS_DISABLED) {
       first_socket = rtc::SSLAdapter::Create(first_socket);
+      ((rtc::SSLAdapter*)first_socket)->set_ignore_bad_cert(true);
```

Comment:
@willkelleher : confirmed I can run against real servers w/o this present... looking into conditional omission.